README.md

```diff
@@ -157,6 +157,22 @@ To:
 
 Also removes trailing commas from object types.
 
+### Optional properties
+
+```js
+/**
+ * @type {{a: number, b?: string}}
+ */
+```
+
+To:
+
+```js
+/**
+ * @type {{a: number, b: string}}
```

Comment:
Did you try if `@type {{a: number, b: string=}}` would work in JSDoc? It would be nice if we could keep the information that `b` is meant to be optional.

Reply:
TS doesn't allow this syntax

<img width="751" alt="Screenshot 2025-05-25 at 11 43 17" src="https://github.com/user-attachments/assets/b0e101a2-95e1-4adb-a4d4-4bf338043287" />

Reply:
`@type {{a: number, b: string=}}` ends up as `Object` in the generated docs.

You'd have to define it as:

```js
/**
 * @type {object}
 * @property {number} a
 * @property {string=} b
 */
```

To output docs for each property and whether or not they're optional